babi/screen.py

```diff
@@ -37,7 +37,9 @@
 # TODO: find a place to populate these, surely there's a database somewhere
 SEQUENCE_KEYNAME = {
     '\x1bOH': b'KEY_HOME',
+    '\x1b[1~': b'KEY_HOME',
     '\x1bOF': b'KEY_END',
+    '\x1b[4~': b'KEY_END',
```

Comment:
```suggestion
    '\x1bOF': b'KEY_END',
    '\x1b[1~': b'KEY_HOME',
    '\x1b[4~': b'KEY_END',
```

I'd order them like this so it keeps the two types together (rather than the two keys together) -- v minor